build.gradle

```diff
@@ -33,7 +33,7 @@ buildscript {
         guavaVersion = '20.0'
         protobufGradlePluginVerison = '0.8.0'
 
-        spineVersion = '0.7.16-SNAPSHOT'
+        spineVersion = '0.7.17-SNAPSHOT'
```

Comment:
Please browse the active branches. I think 0.7.17-SNAPSHOT may already be used. Probably, you'll have to increase the version even more.